WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -312,6 +355,12 @@ - (void)showDetailViewForSubsection:(BlogDetailsSubsection)section
     NSIndexPath *indexPath = [self indexPathForSubsection:section];
 
     switch (section) {
+        case BlogDetailsSubsectionDomainCredit:
+            self.restorableSelectedIndexPath = indexPath;
+            [self.tableView selectRowAtIndexPath:indexPath
+                                        animated:NO
+                                  scrollPosition:[self optimumScrollPositionForIndexPath:indexPath]];
+            break;
```

Comment:
What do you think if we group these 2 cases?
```
case BlogDetailsSubsectionDomainCredit:
case BlogDetailsSubsectionQuickStart:
self.restorableSelectedIndexPath = indexPath;
[self.tableView selectRowAtIndexPath:indexPath
                            animated:NO
                      scrollPosition:[self optimumScrollPositionForIndexPath:indexPath]];
break;
```

Reply:
yup great suggestion! just updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/11537/commits/b960ded43cb0019f231860ef8ab515d8885a2b11